### PR TITLE
Sending generated ships to backend

### PR DESCRIPTION
**Pull request by RafalJDev**

### CODE REVIEW

**Comment by gacikjakub** [comment]
Build 23 Fixed
Running instance of this is on: mergingForPR5.battleships.kuba005.nazwa.pl

File: backend/src/main/java/pl/krkteam/config/WebSecurityConfig.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  }
+
+  @Bean
+  CorsConfigurationSource corsConfigurationSource() {
+    UrlBasedCorsConfigurationSource source = new
+       UrlBasedCorsConfigurationSource();
+    source.registerCorsConfiguration("/**", new CorsConfiguration().applyPermitDefaultValues());
+    return source;
+  }
+}
```
Comment (JaroslawZaczyk) [inline backend/src/main/java/pl/krkteam/config/WebSecurityConfig.java:29]: new line

File: backend/src/main/java/pl/krkteam/controllers/ShipController.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+    ShipFromFronted shipFromFronted = gson.fromJson(post, ShipFromFronted.class);
+
+    System.out.println(shipFromFronted);
+
+    return "{\"message\": \"Cry Germoney!\"}";
+  }
+
+  @PostMapping(value = "/ships")
+  public String shipsFromAngular(@RequestBody String post) {
```
Comment (JaroslawZaczyk) [inline backend/src/main/java/pl/krkteam/controllers/ShipController.java:29]: what is this? change the name.

File: backend/src/main/java/pl/krkteam/model/Ship.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+  public Coordinate[] coordinates;
+
+  @Override
+  public String toString() {
+    return "Ship{" +
+       "coordinates=" + Arrays.toString(coordinates) +
+       '}';
+  }
+}
```
Comment (JaroslawZaczyk) [inline backend/src/main/java/pl/krkteam/model/Ship.java:15]: where are new line signs?

File: backend/src/main/java/pl/krkteam/model/Ships.java [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  @Override
+  public String toString() {
+    return "Ships{" +
+       "ships=" + Arrays.toString(shipArray) +
+       '}';
+  }
+}
+
+
+
```
Comment (JaroslawZaczyk) [inline backend/src/main/java/pl/krkteam/model/Ships.java:18]: too many new lines ;x

File: frontend/src/app/models/board/row/cell/content/content.ts [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+//
+// }
+//
+// export class ContentSunk implements Content{
+//
+// }
+//
+// export class ContentHitted implements Content{
+//
+// }
```
Comment (JaroslawZaczyk) [inline frontend/src/app/models/board/row/cell/content/content.ts:17]: comments

File: frontend/src/app/views/boards/board/board.component.ts [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+    //
+    //   this.calledFunction = setInterval(() => {
+    //     console.log("Callback my dear");
+    //     this.counter++;
+    //     console.log(this.counter);
+    //     if (this.counter == 5) {
+    //       clearInterval(this.calledFunction);
+    //     }
+    //   }, 1200);
+    // }
```
Comment (JaroslawZaczyk) [inline frontend/src/app/views/boards/board/board.component.ts:39]: another comments

File: backend/src/main/java/pl/krkteam/controllers/HelloController.java [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+
+@CrossOrigin(origins = "http://localhost:8080")
+@RestController
+public class HelloController {
+
+  @GetMapping(value = "hello", produces = MediaType.APPLICATION_JSON_VALUE)
+  public String sayHello() {
+    System.out.println("what");
+
+    return "{\"message\": \"Hello, Tylde!\"}";
```
Comment (JaroslawZaczyk) [inline backend/src/main/java/pl/krkteam/controllers/HelloController.java:14]: you've got GSON in your project. Let's use it.

File: backend/src/main/java/pl/krkteam/controllers/HelloController.java [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+  }
+
+  @PostMapping(value = "hello/postt")
+  public String post(@RequestBody String post) {
+    System.out.println(post);
+    return "{\"message\": \"Cry Argentina!\"}";
```
Comment (JaroslawZaczyk) [inline backend/src/main/java/pl/krkteam/controllers/HelloController.java:20]: the same

**Pull request closed without merging**